### ISSUE TITLE
Spawning Vehicles in o

### ISSUE THREAD

**Issue by ahoylad** [pierre-josselin/payday-2-ultimate-trainer-6#89]

**Comment by pierre-josselin** [comment]
Hi, in fact it is not possible to spawn vehicles or any other units if you are not the host of the game.

Also, the button should be disabled if you are not the host, isn't that the case?

**Comment by ahoylad** [comment]
> Hi, in fact it is not possible to spawn vehicles or any other units if you are not the host of the game.
> 
> Also, the button should be disabled if you are not the host, isn't that the case?

Yeah the spawn vehicle button isn't disabled at least for me, and I can try to spawn one although it results in a crash.

**Comment by pierre-josselin** [comment]
Thank you for the feedback, I will fix that!
